helpers/lib_tasks_docker_release.py

```diff
@@ -62,6 +62,251 @@ def _get_dev_version(version: str, container_dir_name: str) -> str:
     return dev_version
 
 
+def _create_multiarch_builder(
+    ctx: Any,
+) -> None:
+    """
+    Create a multi-arch builder for Docker buildx.
+
+    :param ctx: invoke context
+    """
+    # Create a multi-arch builder.
+    platform_builder_name = "multiarch_builder"
+    cmd = rf"""
+    docker buildx rm {platform_builder_name}
+    """
+    # We do not abort on error since the platform builder might be present
+    # or not from previous executions.
+    hsystem.system(cmd, abort_on_error=False)
+    cmd = rf"""
+    docker buildx create \
+        --name {platform_builder_name} \
+        --driver docker-container \
+        --bootstrap \
+        && \
+        docker buildx use {platform_builder_name}
+    """
+    hlitauti.run(ctx, cmd)
+
+
+def _build_multi_arch_image(
+    ctx: Any,
+    opts: str,
+    multi_arch: str,
+    build_args: str,
+    build_image: str,
+    dockerfile: str,
+) -> None:
+    """
+    Build a multi-architecture Docker image.
+
+    :param ctx: invoke context
+    :param opts: build options (e.g., --no-cache)
+    :param multi_arch: target architectures to build for (e.g.,
+        `linux/amd64`, `linux/arm64`)
+    :param build_args: build arguments for the Docker build command
+    :param build_image: name of the image to build
+    :param dockerfile: path to the Dockerfile to use for building
+    """
+    # Build the multi-arch image.
+    # Compress the current directory (in order to dereference symbolic
+    # links) into a tar stream and pipes it to the `docker build` command.
+    # See HelpersTask197.
+    cmd = rf"""
+    tar -czh . | DOCKER_BUILDKIT={DOCKER_BUILDKIT} \
+        time \
+        docker buildx build \
+        {opts} \
+        --push \
+        --platform {multi_arch} \
+        {build_args} \
+        --tag {build_image} \
+        --file {dockerfile} \
+        -
+    """
+    hlitauti.run(ctx, cmd)
+
+
+def _list_image(ctx: Any, image: str) -> None:
+    """
+    List Docker image.
+
+    :param ctx: invoke context
+    :param image: docker image reference in REPOSITORY[:TAG] format
+        Examples:
+        - `*****.dkr.ecr.us-east-1.amazonaws.com/amp:dev-1.0.0`
+        - `*****.dkr.ecr.us-east-1.amazonaws.com/amp:dev`
+        - `sorrentum/cmamp:dev-1.0.0`
+        - `ghcr.io/cryptokaizen/cmamp:prod`
+    """
+    cmd = f"docker image ls {image}"
+    hlitauti.run(ctx, cmd)
+
+
+def _run_tests(
+    ctx: Any,
+    stage: str,
+    version: str,
+    *,
+    skip_tests: Optional[bool] = False,
+    fast_tests: Optional[bool] = False,
+    slow_tests: Optional[bool] = False,
+    superslow_tests: Optional[bool] = False,
+    qa_tests: Optional[bool] = False,
```

Comment:
For me it's kind of weird the the all test are skipping by default. 
Let's make it `True` by default, except the `skip_tests` that I'll keep as `False`.

Reply:
Change all the calls accordingly.